src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+}
+
+.big-bar,
+.small-bar {
+  background: #fff;
+  position: relative;
+  margin-top: 20px;
+}
+
+.bigger {
+  line-height: 68px;
+  padding: 0 0 0 62px;
+}
+
+.bigger:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.smallest {
+  line-height: 40px;
+  padding: 0 0 0 33px;
+}
+
+.smallest:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-input {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  flex: 1;
+  outline: none;
+}
+
+.bigger:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir-Heavy, serif;
+  font-size: 16px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.smallest:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir-Heavy, serif;
+  font-size: 16px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
```

Comment:
You have a doubling code. Create general class and different params take out separately